util/repr-bytes/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "mc-util-repr-bytes"
+version = "0.1.0"
+authors = ["MobileCoin"]
+edition = "2018"
+readme = "README.md"
+
+[dependencies]
+generic-array = "0.13"
+prost = { version = "0.6.1", optional = true, default-features = false, features = ["prost-derive"] }
+serde = { version = "1.0", optional = true, default-features = false, features = ["alloc"] }
+
+[features]
+default = ["alloc", "prost", "serde"]
+alloc = []
```

Comment:
i think that doesn't work -- it will mean that turning on alloc at all forces serde to come on
instead what i tried to do is, put the alloc feature in the `serde = ...` part

Reply:
because cargo doesn't let you do stuff like, make some feature contingent on two other features coming on. and serde here is itself optional

Reply:
Ugh, yeah, OK.

Reply:
maybe serde and prost should fill in alloc? but i don't know how I can express that, cargo tells that a feature and a library cannot have the same name.